src/components/POLine/POLineForm.test.js

```diff
@@ -6,20 +6,23 @@ import { Form } from 'react-final-form';
 import user from '@testing-library/user-event';
 import { render, screen, waitFor } from '@testing-library/react';
 
-import { HasCommand } from '@folio/stripes/components';
 import {
-  ORDER_TYPES,
-  useAccordionToggle,
-} from '@folio/stripes-acq-components';
+  collapseAllSections,
+  expandAllSections,
+  HasCommand,
+} from '@folio/stripes/components';
+import { ORDER_TYPES } from '@folio/stripes-acq-components';
 
 import POLineForm from './POLineForm';
 import { order } from '../../../test/jest/fixtures';
 
 jest.mock('@folio/stripes-acq-components/lib/AcqUnits/AcqUnitsField', () => {
   return () => <span>AcqUnitsField</span>;
 });
-jest.mock('@folio/stripes-acq-components/lib/hooks/useAccordionToggle', () => ({
-  useAccordionToggle: jest.fn().mockReturnValue([jest.fn(), {}, jest.fn()]),
+jest.mock('@folio/stripes-components', () => ({
+  ...jest.requireActual('@folio/stripes-components'),
```

Comment:
Please fix the path to @folio/stripes/components

Reply:
Thank you @Dmytro-Melnyshyn! Updated